From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: nlsdt <nlsdt@nlsdt.cc>
Date: Tue, 5 Aug 2026 00:00:00 +0800
Subject: [PATCH] Fix file_picker git dependency

The git dependency on bggRGjQaUbCoE/flutter_file_picker is no longer
available (404). Replace it with the upstream pub.dev package, matching
the upstream PiliPlus fix.

---
 pubspec.yaml | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/pubspec.yaml b/pubspec.yaml
--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -84,10 +84,7 @@
   extended_nested_scroll_view:
     git:
       url: https://github.com/bggRGjQaUbCoE/extended_nested_scroll_view.git
       ref: mod
-  file_picker:
-    git:
-      url: https://github.com/bggRGjQaUbCoE/flutter_file_picker.git
-      ref: dev
+  file_picker: ^12.0.0-beta.7
   fixnum: ^1.1.1
   fl_chart: ^1.0.0
   flex_seed_scheme: ^4.0.1
-- 
2.44.0
